README.md

```diff
@@ -22,6 +22,10 @@ This project exposes *invalidateFields*: a generator for a [`mutate.options.upda
 
 In some cases after a mutation you want to invalidate cache on other queries that might have become outdated, but you can't really update their results from the data provided by the mutation. The *refetchQueries* is often the tool of choice, but it allows no deep field invalidation, meaning you'll have to invalidate the exact and very specific performed queries. *invalidateFields* is an alternative.
 
+In a nutshell it helps delete recent apollo queries (or just apollo data) from the apollo redux store:
```

Comment:
Apollo doesn't have redux store anymore (and I don't see any connection to it in this project) :wink: